docker-compose/7.1.N-docker-compose.yml

```
@@ -12,7 +12,7 @@ version: "2"
 
 services:
   alfresco:
-    image: quay.io/alfresco/alfresco-content-repository:7.1.1.3
+    image: quay.io/alfresco/alfresco-content-repository:7.1.2-8269
```

not sure which is the versioning scheme with the old 7.1 but I would say `-dddd` are dev versions, so latest available should be:

```suggestion
    image: quay.io/alfresco/alfresco-content-repository:7.1.1.5
```